source/OutfitterPanel.h

```diff
@@ -88,6 +88,10 @@ class OutfitterPanel : public ShopPanel {
 	std::set<Ship *> previousShips;
 	
 	Sale<Outfit> outfitter;
+	
+	// Keep track of how many of the outfitter help screens have been shown
+	int outfithelp = 0;
+	bool checkedhelp = false;
```

Comment:
Should be `checkedHelp`, to match variable naming style.

Reply:
Changed. Thanks!